python/GafferSceneUI/TransformToolUI.py

```diff
@@ -101,6 +109,23 @@ def _distance( ancestor, descendant ) :
 
 	return result
 
+class _TargetTipWidget( GafferUI.Frame ) :
+
+	def __init__( self, tool, **kw ) :
+
+		GafferUI.Frame.__init__( self, borderWidth = 4, **kw )
+
+		labels = Gaffer.Metadata.value( tool, "ui:tool:transformToolTips" ) or []
```

Comment:
We have `tool:` for stuff that applies to all tools - I think `transformTool:toolTips` might sit better with that.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5558/commits/174cef119594ecc1ccec5d8ad7c0c9ddd4b1f063